test/actor.cc

```diff
@@ -7,10 +7,6 @@
 
 #include "test.h"
 
-#if !defined(VALHALLA_SOURCE_DIR)
-#define VALHALLA_SOURCE_DIR
-#endif
-
```

Comment:
we dont have to have this random hack in there and we always get the right value (i have no idea why this kind of thing was hacked in here..)